python/tskit/trees.py

```diff
@@ -2853,6 +2853,38 @@ def b1_index(self):
                 total += 1 / max_path_length[u]
         return total
 
+    def b2_index(self, base):
```

Comment:
Let's default base=10, and put in a paragraph justifying this decision in the docstring.

Reply:
Good idea, I did that.